docs/problem_formulation/problem_formulation_ocp_mex.tex

```diff
@@ -631,47 +686,54 @@ \section{Solver \& Options}\label{sec:solver}
         \multicolumn{4}{l}{\emph{Other}} \\
         \code{print\_level} & int $\geq 0$ & $0$ & verbosity of the solver: ($0$) silent, ($>0$) print first QP problems and solution during SQP\\
         \bottomrule
-    \end{tabulary}
+    \end{tabular}
+    \caption{Solver options that can be set as attributes/properties of \code{AcadosOcpOptions}.} \label{tab:solver_options}
 \end{table}
 %
 \begin{table}
     \centering
-    \caption{Solver \code{set(\str{qp\_solver}, <stringid>)} options. The availability depends on for which solver interfaces \acados{} was linked to.}\label{tab:solver_options:qp_solver_option}
-    \begin{tabular}{c@{ }c@{}c}
+      \begin{tabular}{cccc}
         \toprule
-        Solver lib & Condensing & {String identifier} \\ \midrule
-        \multirow{2}{1.2cm}{\centering HPIPM} & partial & \code{partial\_condensing\_hpipm}*\\
-         & full & \code{full\_condensing\_hpipm} \\%[1ex]
-        HPMPC & partial & \code{partial\_condensing\_hpmpc} \\%[1ex]
-        % \multirow{2}{2cm}{\centering OOQP} & partial & \code{partial\_condensing\_ooqp} \\
+        Solver lib & Condensing & {Attribute/Property} & Reference \\ \midrule
+        \multirow{2}{1.2cm}{\centering HPIPM} & partial & \code{PARTIAL\_CONDENSING\_HPIPM}* & \cite{Frison2020a}\\
+         & full & \code{full\_condensing\_HPIPM} & \cite{Frison2020a}\\%[1ex]
+        HPMPC & partial & \code{PARTIAL\_CONDENSING\_HPMPC} & \cite{Frison2015a}\\%[1ex]
+        % \multirow{2}{2cm}{\centering OOQP} & partial & \code{PARTIAL\_CONDENSING\_ooqp} \\
          % & full & \code{full\_condensing\_ooqp} \\[1ex]
-        OSQP & partial & \code{partial\_condensing\_osqp} \\
+        OSQP & partial & \code{PARTIAL\_CONDENSING\_OSQP} & \cite{Stellato2020} \\
         % QORE & full & \code{full\_condensing\_qore} \\
-        qpDUNES & partial & \code{partial\_condensing\_qpdunes} \\
-        qpOASES & full & \code{full\_condensing\_qpoases} \\
-        DAQP & full & \code{full\_condensing\_daqp} \\
+        qpDUNES & partial & \code{PARTIAL\_CONDENSING\_QPDUNES} & \cite{Frasch2015}\\
+        qpOASES & full & \code{full\_condensing\_QPOASES} & \cite{Ferreau2014}\\
```

Comment:
Capitalization is off, but we could remove the Table and just link to the property:
https://docs.acados.org/python_interface/index.html#acados_template.acados_ocp_options.AcadosOcpOptions.qp_solver

Reply:
capitalization is fixed! I would keep the table to also give the references for the QP solvers